tools/misbehaving-jmx-server/src/main/java/org/datadog/misbehavingjmxserver/DynamicMBeanMetrics.java

```diff
@@ -0,0 +1,177 @@
+package org.datadog.misbehavingjmxserver;
+
+import javax.management.*;
+import javax.management.openmbean.CompositeData;
+import javax.management.openmbean.CompositeDataSupport;
+import javax.management.openmbean.CompositeType;
+import javax.management.openmbean.OpenDataException;
+import javax.management.openmbean.OpenType;
+import javax.management.openmbean.SimpleType;
+import javax.management.openmbean.TabularData;
+import javax.management.openmbean.TabularDataSupport;
+import javax.management.openmbean.TabularType;
+
+import java.lang.management.ManagementFactory;
+import java.util.HashMap;
+import java.util.Map;
+import lombok.extern.slf4j.Slf4j;
+
+@Slf4j
+public class DynamicMBeanMetrics implements DynamicMBean {
+    public final String name;
+    private Map<String, Object> attributes = new HashMap<>();
+    private final MetricDAO metricsDAO;
+    private int numCompPerTabular;
+
+    public DynamicMBeanMetrics(final String name, int numAttributes, int numTabulars, int numCompPerTabular, final MetricDAO metricsDAO) {
+        this.name = name;
+        this.metricsDAO = metricsDAO;
+        this.numCompPerTabular = numCompPerTabular;
+
+        // Add dummy attributes during object initialization
+        for (int i = 1; i <= numAttributes; i++){
+            attributes.put("Attribute" + String.valueOf(i), getSimpleAttributeValue());
+        }
+
+        for (int i = 1; i <= numTabulars; i++) {
+            try {
+
+                attributes.put("Tabular" + String.valueOf(i),getTabularData());
+            } catch (Exception e) {
+                log.warn("Tabular" + String.valueOf(i) + " didnt work");
```

Comment:
```suggestion
                log.warn("Tabular" + String.valueOf(i) + " threw an exception: ", e);
```
and then no need for the below `printStackTrace`